blocks/api/test/parser.js

```diff
@@ -97,11 +98,11 @@ describe( 'block parser', () => {
 		it( 'returns false is block is not valid', () => {
 			registerBlockType( 'core/test-block', defaultBlockSettings );
 
-			expect( isValidBlock(
+			expect( () => isValidBlock(
 				'Apples',
 				getBlockType( 'core/test-block' ),
 				{ fruit: 'Bananas' }
-			) ).toBe( false );
+			) ).toThrow();
```

Comment:
Is a pain to test with the ANSI escapes, unless we were to disable that for testing, or just test the pattern.

>when we're ready to expose this info to the UI?

Which I'm starting to wonder, do we want the error message exposed in the UI anywhere? Currently we're only generating and throwing the error in development modes, which means it's not something generally accessible in the UI (in distributed plugin anyways).